qpc/scan/utils.py

```diff
@@ -45,6 +50,34 @@ def _get_source_ids(parser, source_names):
     return not_found, source_ids
 
 
+def _get_scan_object_id(parser, name):
```

Comment:
I would name this function get_scan_object_id because it is used outside of the util module.